packages/package-utils/src/packageUtils.ts

```diff
@@ -17,18 +17,12 @@ import { runCommand } from './processUtils';
 type PackageName = 'npm' | 'yarn';
 
 export function getPackageManager(): PackageName {
-    const hasLocalYarn = fs.existsSync(path.resolve(process.cwd(), 'yarn.lock'));
```

Comment:
I'd prefer to keep the logic as it is: it's better to check a local file than running a sub process